src/time_tracker/projects/db.clj

```diff
@@ -0,0 +1,81 @@
+(ns time-tracker.projects.db
+  (:require [clojure.java.jdbc :as jdbc]
+
+            [time-tracker.db :as db]))
+
+(defn retrieve-project-if-authorized
```

Comment:
I'd actually drop the word `project` from the functions in this namespace, since it's implicit, and quite expressive when called: `db-project/retrieve-if-authorized`.